src/Symfony/Component/Cache/Adapter/ApcuAdapter.php

```diff
@@ -92,7 +92,11 @@ protected function doDelete(array $ids)
     protected function doSave(array $values, $lifetime)
     {
         try {
-            return array_keys(apcu_store($values, null, $lifetime));
+            if (false === $failures = apcu_store($values, null, $lifetime)) {
```

Comment:
Yes ... we decided a long ago (in 2011 if I remember correctly) to use Yoda-style conditions. I don't think we can change this now, because it will be a nightmare for our mergers.